ADALiOS/ADALiOS/ADAuthenticationContext+TokenCaching.m

```diff
@@ -36,21 +37,12 @@ - (ADTokenCacheStoreItem*)extractCacheItemWithKey:(ADTokenCacheStoreKey*)key
     }
     
     ADAuthenticationError* localError;
-    ADTokenCacheStoreItem* item = [self.tokenCacheStore getItemWithKey:key userId:userId.userId error:&localError];
+    ADTokenCacheItem* item = [self.tokenCacheStore getItemWithKey:key userId:userId.userId error:&localError];
     if (!item && !localError && userId)
-    {//ADFS fix, where the userId is not received by the server, but can be passed to the API:
-        //We didn't find element with the userId, try finding an item with nil userId:
-        NSArray* items = [self.tokenCacheStore getItemsWithKey:key error:&localError];
-        if(items.count) {
```

Comment:
It's still there. I'm taking advantage of the fact that if we don't have an idtoken for an item we cache it with an empty string (@"") as the userid so we can retrieve only the item we care about.